src/main/java/com/notably/model/suggestion/SuggestionModelImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.notably.model.suggestion;
+
+import java.util.List;
+import java.util.Optional;
+
+import javafx.beans.property.Property;
+import javafx.collections.ObservableList;
+
+/**
+ * The implementation class of SuggestionModel.
+ */
+public class SuggestionModelImpl implements SuggestionModel {
+    private ObservableList<SuggestionItem> suggestions;
+    private Property<Optional<String>> responseTextProperty;
+
+    public SuggestionModelImpl(ObservableList<SuggestionItem> suggestions,
+                               Property<Optional<String>> responseTextProperty) {
+        this.suggestions = suggestions;
+        this.responseTextProperty = responseTextProperty;
+    }
```

Comment:
Actually, rather than accepting `suggestions` and `responseTextProperty` as parameters, I believe it's more useful to just make a no-arg constructor that initializes these fields to empty. In a way, your `SuggestionModel`'s initial state is always no suggestion, right!

Reply:
so do I just set
```
public SuggestionModelImpl() {
  this.suggestions = null;
  this.responseTextProperty = null;
}
```
?